server/src/main/java/com/linecorp/centraldogma/server/storage/repository/DiffOption.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.centraldogma.server.storage.repository;
+
+import com.linecorp.centraldogma.common.Change;
+
+/**
+ * The options for diffing a repository.
+ */
+public enum DiffOption {
```

Comment:
Suggestion) `DiffOption` sounds like how to compare two revisions such as `ignore white spaces`. However, this option is used to specify the result style of changed files.
```suggestion
public enum DiffResultType {

   PATCH,
   
   RAW_DATA
```

Reply:
I like the names. 👍

Reply:
I realized that the usage is different. I'll think about it more.

Reply:
I'm okay to leave it as it is because it is only used at the server side which we can easily change.

Reply:
Alright, let me use `DiffResultType` though. 😉